macos-setup.md

```diff
@@ -0,0 +1,51 @@
+## Developer's Guide for Setting Up MacOS Environment
+
+Ensure you have the following prerequisites downloaded:
+
+* `Java` --> [Download Here](https://www.oracle.com/java/technologies/downloads/)
+* `MySQL` --> [Download Here](https://dev.mysql.com/downloads/mysql/)
+
+Once those are downloaded and you have cloned the Git repository, you will want to go into the SpeechTranscription directory and create/activate a Python virtual environment:
+
+* `python3 -m venv venv` to create the environment.
```

Comment:
can add a comment that the specific version of python might need to be included in this command